creme/sketch/apps.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+################################################################################
+#    Creme is a free/open-source Customer Relationship Management software
+#    Copyright (C) 2019-2022  Hybird
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+
+from django.utils.translation import pgettext_lazy
+
+from creme.creme_core.apps import CremeAppConfig
+
+
+class CremeSketchConfig(CremeAppConfig):
+    default = True
+    name = "creme.sketch"
+    verbose_name = pgettext_lazy("sketch", "Sketch")
+    dependencies = ["creme.creme_core"]
+
+    def register_bricks(self, brick_registry):
+        from . import bricks
+
+        brick_registry.register(bricks.DemoChartBrick)
```

Comment:
I'm not fan to expose this brick to common end users (it appears in the bricks configuration in a vanilla install). Why not a setting (False by default) which controls the registration of the brick ? or you plan to remove the registration as soon as "reports" & "graphs" use 'sketch' ?

Reply:
I guess @mrjmad will be able to register this himself if needed.

Reply:
Hum OK but you left the registration....